packages/synapse-react-client/src/components/SynapseHomepage.tsx

```diff
@@ -69,7 +80,12 @@ export const SynapseHomepage: React.FunctionComponent<SynapseHomepageProps> = ({
           <div className="SignUpButtonContainer">
             {accessToken ? (
               <>
-                <Button href={DASHBOARD_LINK} variant="light">
+                <Button
+                  href={DASHBOARD_LINK}
+                  variant="contained"
+                  color="light"
```

Comment:
Added a new "light" color to MUI theme to handle the white buttons on the homepage